src/API/Trace/AbstractSpan.php

```diff
@@ -27,6 +27,14 @@ final public static function getCurrent(): SpanInterface
         return self::fromContext(Context::getCurrent());
     }
 
+    /** @inheritDoc */
+    final public static function setCurrent(SpanInterface $span): Context
```

Comment:
I think we should keep at most two of `::storeInContext()`, `::setCurrent()` and `::activate()`, they all do very similar things.
My preference would be to keep `::storeInContext()` as is, rename `::activate()` to `::makeCurrent()` (and `::getCurrent()` to `::current()`) for consistency with Java.